src/components/Trade/hooks/useSwapper/useSwapperV2.ts

```diff
@@ -70,6 +75,17 @@ export const useSwapper = () => {
     [assetIds, swapperManager],
   )
 
+  const sellAssetAccountIds = useAppSelector(state =>
+    selectPortfolioAccountIdsByAssetId(state, { assetId: sellAsset?.assetId ?? '' }),
+  )
+  const sellAccountFilter = useMemo(
+    () => ({ accountId: sellAssetAccountIds[0] ?? '' }),
```

Comment:
Self-review: Can we make the accountNumber programmatic instead of hardcoded account 0 ?